src/test/java/net/sf/jabref/logic/layout/format/LatexToUnicodeFormatterTest.java

```diff
@@ -28,9 +29,10 @@ public void testFormatStripLatexCommands() {
     @Test
     public void testFormatTextit() {
         // See #1464
-        assertEquals("text", formatter.format("\\textit{text}"));
+        assertEquals("\uD835\uDC61\uD835\uDC52\uD835\uDC65\uD835\uDC61", formatter.format("\\textit{text}"));
```

Comment:
[d835](http://www.charbase.com/d835-unicode-invalid-character) is the symbol for an invalid character. Is this conversation a bug or indeed the expected outcome?